pkg/sink/sink_test.go

```diff
@@ -186,14 +245,28 @@ func Test_createResource(t *testing.T) {
 		t.Fatalf("Error creating RESTClient: %s", err)
 	}
 	tests := []struct {
-		name        string
-		resource    json.RawMessage
-		wantURLPath string
+		name                   string
+		resource               json.RawMessage
+		eventListenerNamespace string
+		wantURLPath            string
 	}{
 		{
-			name:        "PipelineResource",
-			resource:    json.RawMessage(pipelineResourceBytes),
-			wantURLPath: "/apis/tekton.dev/v1alpha1/namespaces/foo/pipelineresources",
+			name:                   "PipelineResource",
```

Comment:
quick nit: please update the test name, because now there're two named `PipelineResource`. One should probably say that it has a namespace, the other that it does not 👍